exec/csnex/Support.cs

```diff
@@ -0,0 +1,84 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+
+namespace csnex
+{
+    public class Support
+    {
+        static public List<string> Paths;
+
+        public Support(string source_path)
+        {
+            Paths = new List<string>();
+            // Always check current directory;
+            Paths.Add(string.Format(".{0}", Path.DirectorySeparatorChar));
```

Comment:
This seems significantly different from the code in support.cpp. The order of paths should be the same as what is already established. Your code changes the order here with no information about why.

Reply:
Changed order of the load paths.